rest/model/conversions.go

```diff
@@ -167,8 +191,20 @@ func generateConversionCode(generatedConversions map[typeInfo]string, inType, ou
 // how it would be referenced in code
 // for example: github.com/a/b/c.myStruct becomes c.myStruct
 func shortenPackage(pkg string) string {
-	split := strings.Split(pkg, "/")
-	return split[len(split)-1]
+	re := regexp.MustCompile("^([^A-Za-z]*)([A-Za-z]+.*)$")
```

Comment:
Just for my own understanding, can you explain this regexp? I'm confused about in the else case what "prefix" would be, is this applicable to your example in 192 or is this for a different case?

Reply:
the change here is to be able to handle non-alphabetic characters at the start of a package name such as [] or *. The change preserves those while still shortening the package. I also added a test which should make this clearer